Modules/System Tests/Cryptography Management/src/Rfc2898DeriveBytesTest.Codeunit.al

```diff
@@ -0,0 +1,22 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+codeunit 132576 "Rfc2898DeriveBytes Test"
+{
+    Subtype = Test;
+
+    [Test]
+    procedure TestDeriveBytes()
+    var
+        CodRfc2898DeriveBytes: Codeunit Rfc2898DeriveBytes;
+        EncryptedText: Text;
+        ExpectedEncryptedText: Label 'RgIsn9T5fqPK8bsXjzlmWqinRxg=';
+    begin
+        // Encrypt Text 
+        EncryptedText := CodRfc2898DeriveBytes.HashRfc2898DeriveBytes('Test', 'Test123', 23, 10);
+        // Verify Result 
+        IF (EncryptedText <> ExpectedEncryptedText) THEN
+            ERROR('Failed to encrypt text');
```

Comment:
Please, user [Library Assert](https://github.com/microsoft/ALAppExtensions/tree/master/Modules/DevTools/TestFramework/TestLibraries/Assert) :)

Also, I recommend following the Given-When-Then template (see [other tests](https://github.com/microsoft/ALAppExtensions/tree/master/Modules/DevTools/TestFramework/TestLibraries/Assert))